HIRS_Utils/src/main/java/hirs/data/persist/BiosMeasurements.java

```diff
@@ -0,0 +1,185 @@
+package hirs.data.persist;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import hirs.persist.ReferenceManifestManager;
+import hirs.persist.ReferenceManifestSelector;
+import hirs.tpm.eventlog.TCGEventLog;
+import hirs.tpm.eventlog.TpmPcrEvent;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+
+/**
+ * Sub class that will just focus on PCR Values and Events.
+ * Similar to {@link hirs.data.persist.SupportReferenceManifest}
+ * however this is the live log from the client.
+ */
+@Entity
+public class BiosMeasurements extends ReferenceManifest {
```

Comment:
Linux calls the file this relates as binary_bios_measurements so the name BiosMeasurements is understandable, however the term BIOS is outdated and the TCG PC Client PFP refers to data being processed as an Event Log (although there is already an EventLog class). Microsoft refers to it as an Measured Boot Log to make things even more tangled. Perhaps something a little more like TcgMeasurements  or EventLogMeasurements might fit here.

Reply:
Agreed, and updated.